tests/integration/test_scmservice_view.py

```diff
@@ -0,0 +1,61 @@
+from uuid import UUID
+
+import pytest
+
+
+@pytest.mark.django_db
+class TestSCMServiceViewSet:
+
+    def test_list_unauthenticated(self, client, scm_service):
+        response = client.get('/scm-services/')
+        assert response.status_code == 403
+
+    def test_list(self, client, logged_in_user, scm_service):
+        response = client.get('/scm-services/')
+        assert response.status_code == 200
+        parsed = response.json()
+        assert len(parsed) == 1
+        assert parsed[0]['type'] == 'bitbucket'
+        assert parsed[0]['server_url'] == 'www.example.com'
+        UUID(parsed[0]['public_identifier'])  # should not raise
```

Comment:
Why do you not compare the UUID to that of the scm_service?

Reply:
because it's create when you save a new object and I just wanna know if it parses correctly